docs/source/getting_started/active_learning.rst

```diff
@@ -172,9 +172,9 @@ here is that the argument `n_samples` always refers to the total size of the lab
 .. code-block:: Python
 
    # we want a total of 200 images after the first iteration (50 new samples)
-   # this time, we use the CORAL sampler and provide a scorer to the query
-   config = SamplerConfig(n_samples=200, method=SamplingMethod.CORAL, name='al-iteration-1')
-   al_agent.query(sampler_config, scorer)
+   # this time, we use the CORAL selection strategy and provide a scorer to the query
+   selection_config = SelectionConfig(n_samples=200, method=SamplingMethod.CORAL, name='al-iteration-1')
```

Comment:
same here with the `method=...`

Reply:
See https://github.com/lightly-ai/lightly/pull/735#discussion_r841432558